apps/api/v2/src/ee/bookings/2024-08-13/controllers/e2e/team-bookings.e2e-spec.ts

```diff
@@ -322,6 +400,61 @@ describe("Bookings Endpoints 2024-08-13", () => {
           });
       });
 
+      it("should create a phone based booking", async () => {
+        const body: CreateBookingInput_2024_08_13 = {
+          start: new Date(Date.UTC(2030, 0, 8, 15, 0, 0)).toISOString(),
+          eventTypeId: phoneOnlyEventTypeId,
+          attendee: {
+            name: "alice",
+            phoneNumber: "+919876543210",
+            email: "test@test.com",
```

Comment:
Also here - if we are testing a phone based booking shouldn't we remove `email` from the payload?